libraries/abstractions/ble_hal/test/src/iot_test_ble_hal_afqp.c

```diff
@@ -651,6 +654,44 @@ TEST( Full_BLE, BLE_Property_WriteLongCharacteristic )
     }
 }
 
+TEST( Full_BLE, BLE_Property_ReadLongCharacteristic )
+{
+    BLETESTreadAttrCallback_t xReadEvent;
+    BTGattResponse_t xGattResponse;
+    BLETESTconfirmCallback_t xConfirmEvent;
+    BTStatus_t xStatus;
+    uint8_t LongReadBuffer[ bletests_LONG_WRITE_LEN ];
```

Comment:
bletests_LONG_WRITE_LEN - Can we have a separate define for the long reads, like bletests_LONG_READ_LEN 

Also its better to not allocate long read buffer on stack, as it might over flow the stack buffer. Could be a static global variable instead.

Reply:
Sure